src/UiRadioGroup.vue

```diff
@@ -129,17 +128,16 @@ export default {
 
         showHelp() {
             return Boolean(this.help) || Boolean(this.$slots.help);
-        }
-    },
-
-    watch: {
-        selectedOptionValue() {
-            this.$emit('update:modelValue', this.selectedOptionValue);
-            this.$emit('change', this.selectedOptionValue);
         },
 
-        modelValue() {
-            this.selectedOptionValue = this.modelValue;
+        selectedOptionValue: {
```

Comment:
I like that the getter and setter here keep the read/write together, over the separate data + watcher we had before. 👍